packages/directory/addon/components/dir-icon.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * {{dir-icon
+ *    value=value
```

Comment:
Can value be a positional param:
```js
positionalParams: ['value']
```
https://guides.emberjs.com/release/components/passing-properties-to-a-component/#toc_positional-params